ui-core/src/styles/select.css

```diff
@@ -5,11 +5,15 @@
     0px 1px 1px rgba(0, 0, 0, 0.2),
     0px 2px 4px rgba(0, 0, 0, 0.1);
   height: 2.5rem;
-  padding: 0.438rem 0.813rem 0.562rem 0.813rem;
+  padding: 0.438rem 2rem 0.563rem 0.813rem;
   letter-spacing: 0;
   text-align: left;
   line-height: 1.5rem;
   width: 100%;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background: inline('../assets/triangle-up-down.svg') no-repeat right .75rem center;
```

Comment:
Related to this [comment](https://github.com/OpenRailAssociation/osrd/pull/8804#discussion_r1751450325), maybe there is more to add to have the icon also when focused or else.

Reply:
To avoid any other kind of conflict, I preferred to rename the class.